matrix/matrix_class.py

```diff
@@ -188,15 +195,15 @@ def get_cofactor(self, row, column):
             return self.get_minor(row, column)
         return -1 * self.get_minor(row, column)
 
-    def minors(self):
+    def minors(self) -> "Matrix":

```

Comment:
```suggestion
    def minors(self) -> Matrix:
```
This would go better